Frame.cc

```diff
@@ -107,21 +107,23 @@ std::vector<float> Frame::getImageData(CARTA::ImageBounds& bounds, int mip, bool
     if (meanFilter) {
         // Perform down-sampling by calculating the mean for each MIPxMIP block
         auto range = tbb::blocked_range2d<size_t>(0, numRowsRegion, 0, rowLengthRegion);
-        auto loop = [&](const tbb::blocked_range2d<size_t> &r) {
-            for(size_t j = r.rows().begin(); j != r.rows().end(); ++j) {
-                for(size_t i = r.cols().begin(); i != r.cols().end(); ++i) {
+        auto loop = [&](const tbb::blocked_range2d<size_t>& r) {
```

Comment:
This looks like an odd spacing change -- deliberate, or typo?

Reply:
intentional, since the `&` is put before the space in most cases in the rest of the file. However, we should probably employ a proper linting tool like `clang-tidy` to make sure we're consistent with a particular style.

Reply:
I'm happy to defer all pedantic style issues until we can clean up everything at once and start using a commit hook.